packages/coordinator/src/components/MuiIconButton/MuiIconButton.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
+import {
+  IconButton as MaterialIcon,
+  PropTypes,
+  SvgIconTypeMap,
+} from "@material-ui/core";
+import { OverridableComponent } from "@material-ui/core/OverridableComponent";
+import Tooltip from "@material-ui/core/Tooltip";
+
+export enum IconSize {
+  small = "small",
+  medium = "medium",
+}
+
+export enum Icon {
```

Comment:
Is this used?

Reply:
This is the old component and it did use it. as we decided, for now we will keep it.